recipes-pv/pantavisor/pantavisor_019.bb

```diff
@@ -80,6 +84,9 @@ do_install() {
 	install -m 0644 ${WORKDIR}/rev0json ${D}/var/pantavisor/storage/trails/0/.pvr/json
 	install -m 0755 ${WORKDIR}/pantavisor-run ${D}/usr/bin/pantavisor-run
 	install -m 0755 ${WORKDIR}/pantavisor-run ${D}/usr/bin/pantavisor-run
+	if ${@bb.utils.contains('PANTAVISOR_FEATURES','uboot-ab','true','false',d)}; then
```

Comment:
here rather test for the existance of the file than adding a conditional. the conditional from further above would ensure that such file isnt there if the feature is not enabled...

Reply:
done